resources/snakefiles/assemble.smk

```diff
@@ -108,10 +108,11 @@ rule quast:
 
 rule multiqc_assemble:
     input:
-        lambda wildcards: expand("output/assemble/quast/{sample}/report.txt",
+        lambda wildcards: expand("output/assemble/{assembler}/quast/{sample}/report.txt",
+                                 assembler=config['assemblers'],
                                  sample=samples)
     output:
-        "output/assemble/multiqc/multiqc.html"
+        "output/assemble/{assembler}/multiqc/multiqc.html"
```

Comment:
There's a conflict here -- the `expand` function in `input` is running multiqc on *all* the assemblers, but the output is per-assembler. We need to pick one or the other.

I'd maybe suggest doing multiqc for the whole shebang, as the point is comparison. So output should be in `output/assemble/multiqc`, and log can remain the same.